docs/fern/docs/pages/sdk/use-user.mdx

```diff
@@ -1,34 +1,37 @@
 ---
-slug: sdk/use-user
+title: useUser
 ---
 
-`useUser` is a hook that returns the user object if the user is authenticated; otherwise, it returns `null` by default. However, if you pass in `{ or: "redirect" }` or `{ or: "throw" }` as an option, it will redirect to the login page or throw an error respectively when the user is not authenticated.
+# useUser
 
-If you want to learn more about the `User` object, check out the [User](./current-user.mdx) documentation.
+A hook to retrieve the current user information with various options for handling authentication and project matching.
 
-## Default Usage
+## Parameters
 
-Check if the user is authenticated and display the user's name.
-```jsx
-import { useUser } from "@stackframe/stack";
+- `options` (optional): `GetUserOptions` - Options for retrieving user information.
+  - `or` (optional): `'redirect' | 'throw'` - Specifies behavior if user is not authenticated.
+  - `projectIdMustMatch` (optional): `'internal'` - Ensures the user belongs to an internal project.
```

Comment:
This type is incorrect, it should be `string`, not `'internal'`. This type is incorrect, it should be `'redirect' | 'throw' | 'return-null'` (where the third is the default). I believe the AI got confused by the many overloads of this function